docs/formats/_index.md

```diff
@@ -11,7 +11,7 @@ weight: 5
 > providing a unified approach to row, columnar, and human-readable formats. Together these
 > represent a superset of both the dataframe/table model of relational systems and the
 > semi-structured model that is used ubiquitously in development as JSON and by NoSQL
-> data stores.  The Super JSON spec has [a few examples](sup.md#3-examples).
+> data stores.  The Super (SUP) format spec has [a few examples](sup.md#3-examples).
```

Comment:
This is one of the exceptions I described in my opening PR notes. In the couple of docs that have these "tl;dr" sections up top, I wrote out the long format name along with acronym (e.g., "Super (SUP) format") since it was the first such reference in the doc, but did it once again the first time the acronym is mentioned in the main text of the doc as well. I did this because by its very definition a "tl;dr" is optional text (i.e., someone who goes to the page with intent to read the whole doc may skip it entirely or pay minimal attention to it), so I prefer not to rely on the assumption that text in a "tl;dr" section is a "first reference" to an important term in a doc.

Reply:
Can we just remove the `**TL;DR**` and `>` line prefixes so this becomes a regular introductory paragraph?

Reply:
@nwt: Certainly! While I know "TL;DR" has its place for blabbermouths like me that write too much detail [ 😛 ] I've always found it a little informal for official docs. Since you seem to agree, and these command docs read a _little_ like `man` pages that tend to lead with a "Synopsis" section that's TL;DR-like, I went ahead and morphed that leading section on the `super` command doc into a Synopsis section and then did the same treatment on the `super db` doc for consistency.